data/themes/darktable-icons.css

```diff
@@ -23,511 +23,579 @@
     background-image: url('../pixmaps/plugins/darkroom/sharpen.svg');
     background-size: contain;
     background-repeat: no-repeat;
-    padding-left: 18px;
-    margin: 3px 0px;
+    margin-top: 0.13em;
```

Comment:
We need to be careful about optical margins. 0.14em is better I think here. I find the icons a little (optically) too up regardless of toggle buttons beside.